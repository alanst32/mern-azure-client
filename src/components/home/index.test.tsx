import React, { useState } from "react";
import {shallow} from 'enzyme';
import Home from ".";

describe('Home', () => {
    let wrapper;
    const setState = jest.fn();
 
    beforeEach(() => {
        wrapper = shallow(<Home></Home>);
    });

    afterEach(() => {
        jest.clearAllMocks();
    });

    it('Render Home properly', () => {
        expect(wrapper).toMatchSnapshot();
    });
})