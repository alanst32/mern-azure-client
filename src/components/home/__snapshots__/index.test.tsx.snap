// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Render Home properly 1`] = `
<div
  className="makeStyles-home-1"
>
  <StudentForm
    totalStudents={0}
  />
  <StudentTable
    students={Array []}
  />
</div>
`;
